packages/lib-utils/src/app/redux/index.ts

```diff
@@ -25,7 +25,14 @@ type UseReduxStoreResult = {
  * ```
  */
 export const useReduxStore = (): UseReduxStoreResult => {
-  const storeContext = useStore();
+  let storeContext: Store | null = null;
+  try {
+    // It'll always be invoked -- it just might blow up
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    storeContext = useStore();
+  } catch (e) {
+    // TODO: Handle this better...
+  }
```

Comment:
This blows up with or without the provider used in HAC-Core's AppEntry file. Not sure what is happening here. This is how I fixed it locally. This is NOT a solution though.

Reply:
Kinda curious why we need this, I've been running both hac-core and hac-dev locally for some time and didn't see this error before :thinking: